packages/manager/modules/vps/src/upscale/configuration-price/configuration-price.scss

```diff
@@ -0,0 +1,12 @@
+vps-upscale-configuration-price {
+  .vps-upscale-summary-price__price {
```

Comment:
I don't see any class applied in your component, is it still useful ? https://github.com/ovh/manager/pull/2525/files#diff-60d175817534df7f5a90ac0825b899bcR1

Reply:
Translations 🙂

Reply:
@marie, used here : https://github.com/ovh/manager/blob/feat/vps-new-range-upgrade/packages/manager/modules/vps/src/upscale/translations/Messages_fr_FR.json#L4

Reply:
This saddens me ... But I guess we don't have any better solution

Reply:
Yes, the translations should be directly in the file, it would be easier for everyone (us included)